src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,117 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
-
+import classnames from 'classnames';
+import propTypes from 'prop-types';
+import { request } from '../../api/getApi';
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie = ({ addMovie, addErrorMessage, getAddError }) => {
+  const [title, getTitle] = useState('');
+  const [api, getApi] = useState('');
+  const [showErrorMessage, getErrorMessage] = useState(true);
```

Comment:
```suggestion
  const [title, setTitle] = useState('');
  const [movie, setMovie] = useState({});
  const [showErrorMessage, setErrorMessage] = useState(true);
```
ти завантажуєш інофрмацію про фільм. Було б доречніше назвати це полу move, а не api